README.md

```diff
@@ -25,6 +25,18 @@
 * [ml5 -> Unet](https://ml5js.org/reference/api-unet/)
 * [p5js](https://p5js.org/)
 
+<div align="center">
+
+[![forthebadge](https://forthebadge.com/images/badges/open-source.svg)](https://forthebadge.com) [![forthebadge](https://forthebadge.com/images/badges/uses-html.svg)](https://forthebadge.com) [![forthebadge](https://forthebadge.com/images/badges/uses-css.svg)](https://forthebadge.com) [![forthebadge](https://forthebadge.com/images/badges/built-by-developers.svg)](https://forthebadge.com) 
```

Comment:
Looks Good :v: 

Just a little bit of modification required. 

- Replace, `CSS` with `Bootstrap`
- Remove `Winter is Coming`.